initfiles/sbin/backup.sh.in

```diff
@@ -55,7 +63,9 @@ fi
 
 log "${header} Backup will occur every ${interval} hours"
 while true; do
-    sleep ${interval}m
+    # background for non blocking wait
+    sleep ${interval}h &
```

Comment:
So if you don't background sleep, it causes the entire backup.sh script to block.  This means that doing a stop on the backupnode component from hpcc-init won't actually reach the backup.sh script until the sleep has completed.  Since this can be for 24+ hours, I figured out a trick to get past this.  If we background the sleep and add a wait, then we aren't in a blocking state.  And backup.sh can still kill the sleep when it traps a SIGINT/SIGTERM.